src/mixing_fab.h

```diff
@@ -0,0 +1,151 @@
+
+#ifndef CYCAMORE_SRC_MIXING_FAB_H_
+#define CYCAMORE_SRC_MIXING_FAB_H_
+
+#include <string>
+#include "cyclus.h"
+#include "cycamore_version.h"
+
+namespace cycamore {
+  
+  /// MixingFab basically mixes N streams with fixed, static, user-specified
+  /// ratios into a single output stream. The MixingFab has N input inventories:
+  /// one for each streams to be mixed, and one output stream. The supplying of
+  /// mixed material is constrained by available inventory of mixed material
+  /// quantities.
+class MixingFab : public cyclus::Facility {
+#pragma cyclus note {   	  \
+    "niche": "mixing facility",				  \
+    "doc":								\
+    "MixingFab basically mixes N streams with fixed, static, user-specified" \
```

Comment:
SG: this should be indented after a colon.  Also, the word basically is not precise and should be removed.